Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md

```diff
@@ -0,0 +1,54 @@
+---
+versionFrom: 8.0.0
+---
+
+# Site Performance checklist
+
+CPU issues can be many different things! Below I will give some advice on narrowing down the issue and trying to find out what is causing the problems.
+The very first thing to do, is try to find out when the trouble started. The friendly Umbraco SWAT will supply you with a graph of your CPU usage for the past 24 hours. Here you will likely notice a spot where it starts going up.
```

Comment:
Perhaps we should add something about how to get in touch with support?
And I think we should call it "support" and not SWAT, as not every one might know that's the support team 😉

Reply:
SWAT has been changed to Support and information on how to get in touch with us as well.